src/pages/database/empty-state/EmptyState.jsx

```diff
@@ -19,15 +20,14 @@ function EmptyState() {
 
   const dbConfig = useSelector(state => getDbConfigs(state))
   const dbAliasNames = Object.keys(dbConfig)
-  const activeDB = dbAliasNames.find(dbAliasName => {
-    return dbConfig[dbAliasName].enabled
-  })
-
-  if (activeDB) {
-    return <Redirect to={`/mission-control/projects/${projectID}/database/${activeDB}/overview`} />;
+  const lastUsedDb = getLastUsedValues(projectID).db;
+  
+  if (dbAliasNames.length > 0 && lastUsedDb) {
```

Comment:
Check if lastUsedDb is present in the dbAliasNames. Also the preference order should be lastUsedDb -> activeDb -> firstDb